flytepropeller/pkg/controller/config/config.go

```diff
@@ -259,7 +259,8 @@ const (
 type EventConfig struct {
 	RawOutputPolicy           RawOutputPolicy `json:"raw-output-policy" pflag:",How output data should be passed along in execution events."`
 	FallbackToOutputReference bool            `json:"fallback-to-output-reference" pflag:",Whether output data should be sent by reference when it is too large to be sent inline in execution events."`
-	ErrorOnAlreadyExists      bool            `json:"error-on-already-exists" pflag:",Whether to return an error when an event already exists."`
+	// only meant to be overridden for certain node types that have different eventing behavior such as ArrayNode
+	ErrorOnAlreadyExists bool `json:"-"`
```

Comment:
I understand the purpose of this now, thanks for the comment. We're adding a field to a central config object and assuming that such field will only be used in a particular situation. This feels hacky to me, but maybe there's a precedent and good reason to follow this pattern.

Reply:
yup - it definitely is hacky. Looking forward when/if we eventually re-work eventing in propeller